rpm_spec/subpackages/manageiq-ansible-venv

```diff
@@ -4,7 +4,7 @@ Summary: %{product_summary} Ansible Runner Virtual Environment
 %global __requires_exclude ^\/var\/lib\/manageiq\/venv\/bin\/python
 
 Requires: ansible >= 5, ansible < 6
-Requires: python38
+Requires: python39
```

Comment:
I'm not sure if this works or not - I thought ansible 5 required 3.8, but if you can get a build I can verify it.

Reply:
I'm not sure either.  I see this in the ansible RPM spec:
```
%if 0%{?rhel} == 8
# RHEL 8's ansible-core package is built using Python 3.8, which is not the default version.
%define python3_pkgversion 38
BuildRequires:  python%{python3_pkgversion}-rpm-macros
%endif
```
So, it may be using system python in EL9.

Reply:
I will update that and rebuild it based on your comment above